v3/plugins/joaompinto/asciidoctor-vscode/latest/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: joaompinto
+apiVersion: v2
+id: joaompinto.asciidoctor-vscode
+version: latest
+type: VS Code extension
+name: AsciiDoctor
+title: AsciiDoctor
+description: AsciiDoc Support, and code browsing https://marketplace.visualstudio.com/items?itemName=joaompinto.asciidoctor-vscode
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+spec:
+  extensions:
+  - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/joaompinto/vsextensions/asciidoctor-vscode/2.7.6/vspackage
```

Comment:
Is the version here intentionally different? In general, the `latest` version of a plugin is just a bit of an ugly workaround; the `latest` `meta.yaml` should be a copy of the newest version in the registry with `version: latest`.

Reply:
done